BootloaderCommonPkg/Library/SecureBootLib/SecureBootRsa.c

```diff
@@ -71,19 +71,24 @@ DoRsaVerify (
     return RETURN_INVALID_PARAMETER;
   }
 
-  CalculateHash  (Data, Length, SignatureHdr->HashAlg, Digest);
+  Status =CalculateHash  (Data, Length, SignatureHdr->HashAlg, Digest);
+  if (EFI_ERROR(Status)) {
+    return RETURN_UNSUPPORTED;
+  }
 
   if (OutHash != NULL) {
     CopyMem (OutHash, Digest, DigestSize);
   }
 
+  DEBUG ((DEBUG_INFO, "SignType (0x%x) SignSize (0x%x)  SignHashAlg (0x%x)\n", SignatureHdr->SigType, SignatureHdr->SigSize, SignatureHdr->HashAlg));

```

Comment:
it is too long. could you spite it into 2 lines?

Reply:
done